splinterd/src/config/builder.rs

```diff
@@ -12,6 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! ConfigBuilder implementation to construct a finalized Config object.
```

Comment:
References to code entities (like "ConfigBuilder") should be between ticks (`) to show up as code in the documentation.